anime_tracker/src/models/APIHandler.ts

```diff
@@ -0,0 +1,13 @@
+import {  OverviewArray } from "./animeSearch";
+
+export class APIHandler {
+  APICALL: string = "https://api.jikan.moe/v4/";
```

Comment:
**naming:** (not just in this file)

* APICALL? -> maybe better something like **base_url**
* getAnimeSearch? what does that mean? search anime by name?

Reply:
adjusted the url name to base_url as it can get used for every request.

getAnimeSearch is named that way due to the used API request [found here](https://docs.api.jikan.moe/#tag/anime/operation/getAnimeSearch). Should I rename it? The idea would be to have multiple functions in this class, for example a function called getAnimeImage with the api request adjusted to image